<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayWidget</class>
 <widget class="QWidget" name="GamePlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;
font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Last move log:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastMoveLog">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Some log&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of sticks:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="makeMoveButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Move</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>270</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>1st ballot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>20</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>2nd ballot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inFirstBallot">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;</string>
   </property>
   <property name="text">
    <string>&lt;N&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inSecondBallot">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;</string>
   </property>
   <property name="text">
    <string>&lt;N&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>&lt;NAME&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidInputLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>Invalid Input</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
